examples/vision/reptile.py

```diff
@@ -0,0 +1,248 @@
+"""
+Title: Few-Shot learning with Reptile
+Author: [ADMoreau](https://github.com/ADMoreau)
+Date created: 2020/05/21
+Last modified: 2020/05/29
+Description: Reptile meta learning algorithm implemented in Keras
```

Comment:
meta-learning

Also, "implemented in Keras" is not necessary: this is keras.io :)